packages/frontend/core/src/components/affine/page-properties/common.ts

```diff
@@ -1,6 +1,38 @@
-import { createContext } from 'react';
+import { useQuery } from '@affine/core/hooks/use-query';
+import type { PagePropertyType } from '@affine/core/modules/properties/services/schema';
+import { WorkspaceFlavour } from '@affine/env/workspace';
+import { getWorkspacePageMetaByIdQuery } from '@affine/graphql';
+import { createContext, useContext } from 'react';
+import type { Middleware } from 'swr';
 
 import type { PagePropertiesManager } from './page-properties-manager';
 
 // @ts-expect-error this should always be set
 export const managerContext = createContext<PagePropertiesManager>();
+
+export const useCloudPageMeta = (type: PagePropertyType) => {
```

Comment:
@EYHN take a look, do we need a service for it

Reply:
sure i will make a child pr for this pr